### ISSUE TITLE
slow/choppy rendering on slither.io on cr69 build

### ISSUE THREAD

**Issue by LaurenWags** [brave/browser-laptop#15151]
### Description
Testing slither.io on `0.23.203` is slow/choppy on screen as compared to prod build `0.23.105`.

### Steps to Reproduce

  1. Launch 0.23.203.
  2. Navigate to slither.io and play for a minute. 

**Actual result:**
Screen seems slow to catch up/choppy. 

**Expected result:**
Should be smooth as on 0.23.105.

**Reproduces how often:**
easily

### Brave Version

**about:brave info:**
 
Brave: 0.23.203 
V8: 6.9.427.18 
rev: 16c2c198f897b4ffbfdbd3e8f725af7d20f85e75 
Muon: 8.1.2 
OS Release: 16.7.0 
Update Channel: Beta 
OS Architecture: x64 
OS Platform: macOS 
Node.js: 7.9.0 
Tor: 0.3.3.8 (git-99f9816efa08ab83) 
Brave Sync: v1.4.2 
libchromiumcontent: 69.0.3497.72

**Reproducible on current live release:**
no

### Additional Information
reproduced on macOS by @kjozwiak

**Comment by bridiver** [comment]
did you check against Chrome 69 beta?

**Comment by LaurenWags** [comment]
good call @bridiver - I'm seeing the same on Chrome Beta. @kjozwiak can you confirm if you do as well?

**Comment by bridiver** [comment]
Yes, but can you file an upstream issue? Hopefully they will fix before the final 69 release

**Comment by kjozwiak** [comment]
It looks like there was a regression with the canvas performance which is being fixed in https://bugs.chromium.org/p/chromium/issues/detail?id=872117.

**Comment by LaurenWags** [comment]
Verified with macOS 10.12.6 using
* 0.23.205 81396b3
* Muon 8.1.4
* libchromiumcontent 69.0.3497.92
